scripts/patch/webpackSRI.js

```diff
@@ -0,0 +1,11 @@
+const SriPlugin = require('webpack-subresource-integrity');
+
+module.exports = webpackConfig => {
+	webpackConfig.plugins.push(
```

Comment:
It would be trickier if we had to add `webpack` plugins in a certain order, but this one needs to be the last one so we just push it to the `Array` of plugins.